content/40_deploying_applications_to_eks/20_create_a_namespace.md

```diff
@@ -6,12 +6,12 @@ weight = 20
 
 Next, declare a [namespace object](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/). This will scope your objects to a name of your choosing, so that in this workshop you won't accidentally interfere with other participants.
```

Comment:
Are users going to be on a shared cluster? If so, the instructions have them all creating the same namespace since it's explicitly setting the `.metadata.name`

Reply:
Nah, each user will have their own AWS account :)

Reply:
Consider dropping the `ns.metadata` since auto-naming will take over